api/envoy/config/filter/network/thrift_proxy/v2alpha1/route.proto

```diff
@@ -46,6 +47,8 @@ message RouteMatch {
   // Inverts whatever matching is done in match_specifier. Cannot be combined with wildcard matching
   // as that would result in routes never being matched.
   bool invert = 3;
```

Comment:
Since invert doesn't apply to the header matchers, we should probably update this comment to make clear (in the docs) what this inverts. Specifically remove the reference to "match_specifier" since that doesn't show up in the docs.